themes/courses.labster.com/lms/templates/ccx/enrollment.html

```diff
@@ -6,14 +6,15 @@
 
 <h2 class="hd hd-2">${_("Batch Enrollment")}</h2>
 <div class="batch-enrollment" style="float:left;width:50%">
-  <form method="POST" action="ccx_invite">
+  <form method="POST" action="ccx_invite" onsubmit="return ccxInviteForm(this)">
+  <p class="error-message"></p>
   <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
-  <label for="student-ids" class="sr">${_("Email Addresses/Usernames")}</label>
+  <label for="student-ids" class="sr">${_("Email Addresses")}</label>
   <p id="label_student_ids" class="text-helper">
-      ${_("Enter email addresses and/or usernames separated by new lines or commas.")}
+      ${_("Enter email addresses separated by new lines or commas.")}
       ${_("You will not get notification for emails that bounce, so please double-check spelling.")}
   </p>
-  <textarea rows="6" name="student-ids" id="student-ids" aria-describedby="label_student_ids" placeholder="${_("Email Addresses/Usernames")}" spellcheck="false"></textarea>
+  <textarea rows="6" name="student-ids" id="student-ids" aria-describedby="label_student_ids" placeholder="${_("Email Addresses")}" spellcheck="false"></textarea>
```

Comment:
Can we use `<input type="email" />` with `multiple` attribute instead?
See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email#Allowing_multiple_email_addresses

Reply:
@polesye do you mean to change textarea to input field, if yes we can but it requires to change the behavior of the user (user can't separate email with new line ) and also we need to think about the UI for that field.
what do you think?

Reply:
Valid point, but I think that most instructors just copy-pase the data from google sheets or Excel, so it shouldn't be a big problem. But you can clarify it with Emma.